### ISSUE TITLE
PlantEquipmentOperation:OutdoorWetbulb not working properly

### ISSUE THREAD

**Issue by asparke2** [NatLabRockies/EnergyPlus#5153]
@kbenne is adding operation schemes to OS, so I'm testing them out.  I don't think the PlantEquipmentOperation:OutdoorWetbulb object is working correctly in EnergyPlus.

To reproduce:
1. Open the example file CoolingTowerWetBulbRangeOp.idf
2. Modify the PlantEquipmentOperation:OutdoorWetbulb so that the first range (All Towers) is -50C to 15C, and the second range, One Tower, is 15C to 70C

![image](https://cloud.githubusercontent.com/assets/2009290/9530358/7d15dad4-4cbc-11e5-8efd-e9c19e59bbbb.png)
1. Run an annual simulation

If you plot the tower heat transfer rate for both towers vs OA wetbulb, it appears that both towers are running above 15C, where the operation scheme is One Tower.  Since the points below overlap perfectly, I tried to edit the colors and transparencies to make it obvious, but if you graph it yourself it is easier to see.

![image](https://cloud.githubusercontent.com/assets/2009290/9530246/e4065058-4cbb-11e5-90ff-37eb9dda066e.png)

**Comment by asparke2** [comment]
PlantEquipmentOperation:OutdoorDrybulb also appears to be affected.  See graph below from CoolingTowerDryBulbRangeOp.idf with no modifications other than annual simulation.  The operation scheme shows All Towers up to 27C, and One Tower above 27C.

![image](https://cloud.githubusercontent.com/assets/2009290/9530719/bd9d62a0-4cbe-11e5-9a87-61f796d1de3e.png)

And both towers are running above 27C drybulb

![image](https://cloud.githubusercontent.com/assets/2009290/9530664/6fbd476c-4cbe-11e5-9ac0-f0d09143b770.png)

Probably worth checking out the other OA-based condenser loop operation schemes too.

**Comment by mjwitte** [comment]
@asparke2 It could be more general than this.  Does the load range based operation give you only one tower when expected?

**Comment by asparke2** [comment]
I haven't looked at a load-based range with cooling towers, but with chillers and boilers it seems to be working fine:

![image](https://cloud.githubusercontent.com/assets/2009290/9531791/080ad420-4cc5-11e5-8a1b-09ee39921020.png)

**Comment by mjwitte** [comment]
@asparke2 My gut feel is that the operation schemes are sending the right signals, but that towers are not listening.  But I could be wrong.

**Comment by rraustad** [comment]
I think this might do the trick.

at line 2290 of CondenserLoopTowers.cc in Init.

```
if ( FirstHVACIteration || RunFlag ) {
  flowRequest = SimpleTower( TowerNum ).DesWaterMassFlowRate * SimpleTower( TowerNum ).TowerMassFlowRateMultiplier;
} else {
  flowRequest = 0.0;
}
WaterMassFlowRate = RegulateCondenserCompFlowReqOp( SimpleTower( TowerNum ).LoopNum, SimpleTower( TowerNum ).LoopSideNum, SimpleTower( TowerNum ).BranchNum, SimpleTower( TowerNum ).CompNum, flowRequest );
```

**Comment by EnergyArchmage** [comment]
I agree with @mjwitte  here, the single speed tower code doesn't really use the load dispatch singles, only temperature setpoints.  The chillers and boilers are better at responding to myload. 

I agree with @mitchute that #4262 and this #5153 could be the same thing, more than one tower behaving poorly because of how the original tower algorithms are implemented for temperature setpoint.

**Comment by jmarrec** [comment]
Still true in **v9.4.0**. I added a defect file on DevSupport, I took CoolingTowerWetBulbRange, changed the PlantEquipmentOperation:OutdoorWetBulb, added Output:SQLite and set a Runperiod of 1/1 to 12/31.

```
  PlantEquipmentOperation:OutdoorWetBulb,
    Year Round Tower Operation,  !- Name
    -50,                     !- Wet-Bulb Temperature Range 1 Lower Limit {C}
    15,                      !- Wet-Bulb Temperature Range 1 Upper Limit {C}
    All Towers,              !- Range 1 Equipment List Name
    15,                      !- Wet-Bulb Temperature Range 2 Lower Limit {C}
    70,                      !- Wet-Bulb Temperature Range 2 Upper Limit {C}
    One Tower;               !- Range 2 Equipment List Name

  CondenserEquipmentList,
    All Towers,              !- Name
    CoolingTower:SingleSpeed,!- Equipment 1 Object Type
    Big Tower1,              !- Equipment 1 Name
    CoolingTower:SingleSpeed,!- Equipment 2 Object Type
    Big Tower2;              !- Equipment 2 Name

  CondenserEquipmentList,
    One Tower,               !- Name
    CoolingTower:SingleSpeed,!- Equipment 1 Object Type
    Big Tower1;              !- Equipment 1 Name
```

Reproducing Andrew's graph shows exactly the same:

![image](https://user-images.githubusercontent.com/5479063/100760754-6389a780-33f2-11eb-9199-f1ecfaef6218.png)

**Comment by joseph-robertson** [comment]
@asparke2 @lymereJ I think this issue may be resolved (although I haven't pinpointed exactly when/where it was fixed). Here are the plots of CoolingTowerWetBulbRangeOp.idf results for various E+ versions:

**v22.2** (same as v9.4 shown above)

<img width="1902" height="903" alt="Image" src="https://github.com/user-attachments/assets/6c8e5e98-17ed-4cbb-9d41-b0a28c52b76f" />

**v23.1** (same as shown [here](https://github.com/NatLabRockies/EnergyPlus/pull/9727#issuecomment-1299123638) in #9727)

<img width="1900" height="904" alt="Image" src="https://github.com/user-attachments/assets/984ca2d1-e95e-4063-bafa-87b8e4e97ba1" />

**v26.1** (only Big Tower1 operates above 15C)

<img width="1902" height="897" alt="Image" src="https://github.com/user-attachments/assets/2f0581bc-1a13-4cc2-a772-2104345a01b8" />

**Comment by mitchute** [comment]
@joseph-robertson You said Big Tower 1 only operates above 15 C, but there are some points below, too. What am I missing?

**Comment by joseph-robertson** [comment]
Sorry, let me say that better. Both Big Tower1 and Big Tower2 can operate below 15C. _Only_ Big Tower1 can (also) operate above 15C.

**Comment by mitchute** [comment]
After a little digging, it looks like https://github.com/natlabrockies/energyplus/pull/10653 fixed this. Closing.
